addons/payment_asiapay/views/payment_provider_views.xml

```diff
@@ -24,6 +24,9 @@
                            groups="base.group_no_one"/>
                 </group>
             </group>
+            <field name="available_currency_ids" position="attributes">
+                <attribute name="required" separator=" or " add="code == 'asiapay' and state != 'disabled'"/>
```

Comment:
`> 100`

```suggestion
                <attribute
                    name="required" separator=" or " add="code == 'asiapay' and state != 'disabled'"
                />
```

Reply:
Also, does it still work if you wrap the content of the `add` attribute inside parentheses? The order of precedence of `and` vs. `or` is not well known in Python, so it'd avoid future headaches should another provider extend this `required` attribute later on.

Reply:
Indeed, I'll add parentheses